tests/email/test_dm_mailchimp.py

```diff
@@ -364,3 +364,74 @@ def test_offset_increments_until_no_members(self):
                     mock.call('a_list_id', count=100, offset=600),
                     mock.call('a_list_id', count=100, offset=700),
                 ]
+
+    def test_get_lists_for_email(self):
+        dm_mailchimp_client = DMMailChimpClient('username', DUMMY_MAILCHIMP_API_KEY, logging.getLogger('mailchimp'))
+        with mock.patch.object(dm_mailchimp_client._client.lists, 'all', autospec=True) as all_lists:
+            all_lists.return_value = {
+                "lists": [
+                    {"id": "gadz00ks", "name": "Pistachios", "irrelevant": "custard"},
```

Comment:
What's the point of these `irrelevant` and `meaningless` keys here?

Reply:
It's to prove that keys other than `id` and `name` don't affect behaviour.

Reply:
Could you use variable names that reflect that, then?

Reply:
`irrelevant`.